tests/docker/.env.sample

```diff
@@ -0,0 +1,2 @@
+# your eth account id
+ACCOUNT_ID=0xb0E31D878F49Ec0403A25944d6B1aE1bf05D17E1
```

Comment:
where'd you get this from? i'd also add the private key somewhere so that it can be used to send txs / deploy stuff etc!

Reply:
This is from a fresh account I made just for this